gcp/utils/count_labels.py

```diff
@@ -0,0 +1,47 @@
+from google.cloud import storage
+import re
+
+# count the number of boxes from a txt string
+def count_from_txt(txt):
+    # labels are stored as integers
+    counts = {}
+    for line in txt.splitlines():
+        l = line.split()
+        if int(l[0]) not in counts.keys():
+            counts[int(l[0])] = 1
+        else:
+            counts[int(l[0])] = counts[int(l[0])] + 1
+    return counts
+
+def main():
+    client = storage.Client()
+    bucket = client.get_bucket('street2sat-model-labeled-data')
+
+    labels_train_prefix = 'run2/labels/train/'
+    labels_val_prefix = 'run2/labels/val/'
```

Comment:
I suggest putting the above few lines in some common util as similar things are used in the other script